cogs/displayinfo.py

```diff
@@ -33,85 +33,113 @@ class DisplayInfo(commands.Cog):
 
     def __init__(self,bot):
         self.bot = bot
-        self.timers = {}
-        self.current_collection = ""
+        self.current_collection = db["UserTracking"] # TODO: Move this secrets file
+        self.timers = self.current_collection.find_one() 
         self.current_guild_name = ""
         self.current_date = str(date.today())
+        self.members_online = set()
+        self.timed_task = None
+    
+    @commands.Cog.listener()
+    async def on_ready(self):
+        for guild in self.bot.guilds:
+            for member in guild.members:
+                if member.status == discord.Status.online:
```

Comment:
Member should also have an activity going for us to start the timed task. No reason to start it if people are just online.